osu.Game/Screens/Edit/Editor.cs

```diff
@@ -235,16 +237,27 @@ public override void OnEntering(IScreen last)
             Beatmap.Value.Track?.Stop();
         }
 
+        private bool isExitingGame = false;
+
         public override bool OnExiting(IScreen next)
         {
             Background.FadeColour(Color4.White, 500);
 
             if (Beatmap.Value.Track != null)
             {
-                Beatmap.Value.Track.Tempo.Value = 1;
-                Beatmap.Value.Track.Start();
+                if (isExitingGame)
+                {
+                    Beatmap.Value.Track.Stop();
```

Comment:
why brackets
```csharp
If (isExitingGame)
    Beatmap.Value.Track.Stop();
else
{
    /* other stuff */
}

Reply:
From what I know, many programmers find it less readable when half of "if-else" statement has brackets and half does not, so if "else" has brackets, "if" should have brackets as well, even if it doesn't need to. Personally I agree with it and I find it more readable, but if you do not like it, I am more than willing to change that.